docs/admin/install.md

```diff
@@ -1,14 +1,20 @@
 # Installing the App in Nautobot
 
+Here you will find detailed instructions on how to **install** and **configure** the App within your Nautobot environment.
+
 ## Prerequisites
 
 - The plugin relies on [`nautobot_plugin_nornir`](https://pypi.org/project/nautobot-plugin-nornir/) to be installed and both plugins to be enabled in your configuration settings.
-- The plugin is compatible with Nautobot 1.4.0 and higher.
+- The plugin is compatible with Nautobot 2.0.0 and higher.
```

Comment:
@snaselj should they both be here? Since it is compatible with older versions as well.

Reply:
There was a note pointing to compatibility matrix below in this chapter. Collapsed it to this line, WDYT @jeffkala?

Reply:
I like this better, thanks.